Services/Twilio/Rest/Addresses.php

```diff
@@ -3,4 +3,19 @@
 class Services_Twilio_Rest_Addresses
     extends Services_Twilio_ListResource
 {
+
+	public function create($customerName, $street, $city, $region, $postalCode, $isoCountry,$friendlyName = null) {
+		$params = array(
+				'CustomerName' => $customerName,
+				'Street' => $street,
+				'City' => $city,
+				'Region' => $region,
+				'PostalCode' => $postalCode,
+				'IsoCountry' => $isoCountry,
+		);
+		if (!is_null($friendlyName)) {
```

Comment:
I was trying to avoid this with the call to empty, but you're right, `0` could be a valid value for `$friendlyName`.